docs/recommendations.md

```diff
@@ -16,3 +16,51 @@ Always bind to localhost rather than to 0.0.0.0 or any interface, unless there i
 Binding to 0.0.0.0 or "all" interfaces can make your application respond on all current and future network interfaces.
 
 Issue: ([#19](https://github.com/open-telemetry/sig-security/issues/19#issue-1926445623))
+
+## Container images
+
+### Use the smallest image possible
+While the size of an image doesn't matter for the runtime, we are responsible for everything that is in the image. If you are delivering a statically linked binary, you can probably build a `scratch` image for production purposes. It might be a good idea to provide a second set of images for debugging purposes, such as ones including a shell and networking utilities.
+
+If you need to include files in the image, such as root certificates, use the builder pattern to obtain the files and copy only those into the final image. Example:
+
+```Dockerfile
+FROM alpine:3.20 as certificates
+RUN apk --no-cache add ca-certificates
+
+FROM scratch
+COPY --from=certificates /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY mybinary mybinary
+```
+
+### Use a non-privileged user
+Under some circumstances, your image might be executed as a privileged user by default. In others, like a hardened Kubernetes distribution, your container is forced to run using a non-privileged account and bugs might appear as the image wasn't built with that in mind. It's recommended to set a high user ID when building the image, to ensure the process always runs as a non-privileged user. This will not only make it more secure by default but will help you uncover issues before they affect security-conscious users.
+
+Here's how to do it:
+
+```Dockerfile
+USER 65532:65532
+```
+
+### Keep your images up to date
```

Comment:
Maybe call out how we're using this in Semconv for *tooling based* docker images:

- The makefile which pulls in dependencies: https://github.com/open-telemetry/semantic-conventions/blob/main/Makefile#L32
- The file which just declares depenencies for dependabot: https://github.com/open-telemetry/semantic-conventions/blob/main/dependencies.Dockerfile